README.md

```diff
@@ -39,7 +39,8 @@ You can then use the following selector anywhere in your project:
 | property | Type  | Description |
 | --- | ---  | --- |
 | options | object | holds all toggle button style configurations |
-| isActive | false | holds the current boolean state of the button - can be `false` or `true` |
+| value | false | holds the current boolean state of the button - can be `false` or `true` |
```

Comment:
- Type should be Boolean
- And in the description you can right that defaults to `false`